.github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml

```diff
@@ -129,4 +129,6 @@ jobs:
         uses: actions/upload-artifact@v3
```

Comment:
a condition

`if: always()`

should make upload step alway run. ref: https://github.com/actions/upload-artifact/issues/328#issuecomment-1159824887